sdk/storage/Azure.Storage.Common/README.md

```diff
@@ -10,55 +10,70 @@ and Azure Queues.
 The Azure.Storage.Common library provides infrastructure shared by the other
 Azure Storage client libraries.
 
-[Source code][source] | [Package (NuGet)][package] | [API reference documentation][rest_docs] | [Product documentation][product_docs]
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
 
 ## Getting started
+
 ### Install the package
+
 Install the Azure Storage client library for .NET you'd like to use with
 [NuGet][nuget] and the common client library will be included:
 
 ```Powershell
-Install-Package Azure.Storage.Blobs
-Install-Package Azure.Storage.Queues
-Install-Package Azure.Storage.Files
+dotnet add package Azure.Storage.Blobs --version 12.0.0-preview.1
+dotnet add package Azure.Storage.Queues --version 12.0.0-preview.1
+dotnet add package Azure.Storage.Files --version 12.0.0-preview.1
 ```
 
-**Prerequisites**: You must have an [Azure subscription][azure_sub], and a
-[Storage Account][storage_account_docs] to use these packages.
+### Prerequisites
 
-To create a Storage Account, you can use the [Azure Portal][storage_account_create_portal],
-[Azure PowerShell][storage_account_create_ps] or [Azure CLI][storage_account_create_cli]:
+You need an [Azure subscription][azure_sub] and a
+[Storage Account][storage_account_docs] to use this package.
+
+To create a new Storage Account, you can use the [Azure Portal][storage_account_create_portal],
+[Azure PowerShell][storage_account_create_ps], or the [Azure CLI][storage_account_create_cli].
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
 
 ## Key concepts
+
 The Azure Storage common client library contains shared infrastructure like
 [authentication credentials][auth_credentials] and [StorageRequestFailedException][StorageRequestFailedException].
 
 ## Examples
-Please see the examples for [Blobs][blobs_examples], [Files][blobs_examples],
-or [Queues][blobs_examples] to get started.
+
+Please see the examples for [Blobs][blobs_examples], [Queues][blobs_examples], and [Files][blobs_examples].

```

Comment:
All these point to the `blob_examples` link

Reply:
Good catch!  Fixed.